IdentityCore/src/cache/token/MSIDCredentialCacheItem+MSIDBaseToken.m

```diff
@@ -53,6 +53,14 @@ - (MSIDBaseToken *)tokenWithType:(MSIDCredentialType)credentialType
         {
             return [[MSIDIdToken alloc] initWithTokenCacheItem:self];
         }
+#if AD_BROKER
+        case MSIDPrimaryRefreshTokenType:
+        {
+            return [[NSClassFromString(@"ADBrokerPrimaryRefreshToken") alloc] initWithTokenCacheItem:self];
```

Comment:
Or we could provide an object to accessors that is responsible for converting cache items to tokens? Let's discuss maybe tomorrow in scrum.

Reply:
Talked to Olga offline, we are going to move `ADBrokerPrimaryRefreshToken` to common core, such that the above issue is gone :) 
Now fixed.